.editorconfig

```diff
@@ -0,0 +1,37 @@
+root = true
+
+[*]
+indent_style = space
```

Comment:
This is only temporary, until we add our linting setup for the whole repo (then we'll switch to tabs). I didn't want to do it now, because it's a huge diff.